rules/windows/command_and_control_rare_microsoft_office_child_processes.toml

```diff
@@ -0,0 +1,38 @@
+[metadata]
```

Comment:
Delete this file before merging!

Reply:
removed: https://github.com/elastic/detection-rules/pull/3281/commits/44209c77528061082d18118d50a6462de2311ba4